aten/src/ATen/cuda/detail/KernelUtils.h

```diff
@@ -28,7 +28,7 @@ constexpr int CUDA_NUM_THREADS = 1024;
 inline int GET_BLOCKS(const int N)
 {
   AT_ASSERTM(N > 0, "CUDA kernel launch blocks must be positive, but got N=", N);
-  return (N + CUDA_NUM_THREADS - 1) / CUDA_NUM_THREADS;
+  return N / CUDA_NUM_THREADS + (N % CUDA_NUM_THREADS == 0 ? 0 : 1);
```

Comment:
Would that be equivalent, considering that it's already be asserted that N > 0?
```suggestion
  return 1 + (N -1 ) / CUDA_NUM_THREADS;
```